### PR TITLE
feature: portfolio api integration and updates 

### PR DESCRIPTION
**Pull request by SorinC6**
task: https://emurgo.atlassian.net/browse/YOEXT-1199

### CODE REVIEW

**Comment by emurgo-rnd** [comment]
[YOEXT-1199: Portfolio API integration](https://emurgo.atlassian.net/browse/YOEXT-1199)

**Review by SorinC6 — COMMENTED** [review]
/check

**Review by SorinC6 — COMMENTED** [review]
/check

File: packages/yoroi-extension/app/UI/features/portfolio/useCases/TokenDetails/OverviewPerformanceDetails/Overview.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  const isPrimary = isPrimaryToken(tokenInfo);
+  return (
+    <Stack direction="column" spacing={theme.spacing(2)}>
+      <Stack direction="row" alignItems="center" spacing={theme.spacing(1)}>
+        <Box
+          width="32px"
+          height="32px"
+          sx={{
+            backgroundColor: theme.palette.ds.gray_300,
+            borderRadius: `50px`,
```
Comment (Nebyt) [inline packages/yoroi-extension/app/UI/features/portfolio/useCases/TokenDetails/OverviewPerformanceDetails/Overview.tsx]:
```suggestion
            borderRadius: `${theme.shape.borderRadius}px`,
```

File: packages/yoroi-extension/app/UI/features/portfolio/useCases/TokenDetails/TokenDetails.tsx [hunk 2]
```diff
@@ -89,84 +51,32 @@ const TokenDetails = ({ tokenInfo }: Props): JSX.Element => {
             cursor: 'pointer',
           }}
```
Comment (Nebyt) [inline packages/yoroi-extension/app/UI/features/portfolio/useCases/TokenDetails/TokenDetails.tsx:52]:
```suggestion
            paddingLeft: '4px',
          }}
```

**Review by SorinC6 — COMMENTED** [review]
/check

**Review by Nebyt — DISMISSED** [review]

**Review by zuzunker — APPROVED** [review]

**Pull request merged**